tensorflow_datasets/image/celebahq.py

```diff
@@ -43,7 +43,8 @@
 High-quality version of the CELEBA
 dataset, consisting of 30000 images in 1024 x 1024 resolution.
 
-WARNING: this dataset currently requires you to prepare images on your own.
+WARNING: This dataset currently requires you to prepare images on your own. Tensorflow also provides an image generator 
```

Comment:
The link you added is to Tensor2Tensor library.

Not sure what you meant by 'provides an image generation'?
That library unfortunately also requires you to prepare the data yourself.

Reply:
I meant to provide a method for those interested in image generation, does it seem unnecessary?

Reply:
I think that it is not necessary.